frontend/src/Components/AlgInputs/AlgInputs.js

```diff
@@ -31,30 +29,7 @@ const useStyles = makeStyles(theme => ({
 
```

Comment:
This was reviewed in initial PR - moved function to helper function for better reusability.